backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -1714,7 +1720,7 @@ public ExperimentRun deepCopyExperimentRunForUser(
           stringQueryBuilder.append(" AND ");
         }
       }
-      Query query = session.createQuery(stringQueryBuilder.toString());
+      Query query = session.createQuery(stringQueryBuilder.toString() + " AND er.deleted = false ");
```

Comment:
Should we use ModelDBConstants.DELETED here too?

Reply:
updated with constant name